### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][NVDA][MAS4.2.1][IntelliJ][Wipro]:NVDA is announicng extra information after invoking "Add" button using shortcut key in Push image window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1500]
**Repro Steps:**

1. Create a new dynamic web project.
2. Navigate to and select Project ->Azure->Push Image.
3. Push image dialog will open.
4. In Push Image Dialog, .Navigate to  "List" which is present under Before launch:Build Artifacts,Activate Tool window then observe the NVDA Behavior

**Expected Result:**
**NVDA Behavior:**
NVDA should announce
Add New Configuration  list
Run External tool  selected  1 of 7

**Actual Result:**
**NVDA Behavior:**
NVDA is announcing as 
sample - IntelliJ IDEA
Push Image  dialog  
Invalid Data: Please specify a valid Docker file.
Artifact
Docker File
Server URL
Username
Password
Image and tag
Container Registry
 /
Before launch: Build Artifacts, Activate tool window
Add New Configuration  list
Run External tool  selected  1 of 7
**Jaws Behavior:** Jaws is announcing as expected
Add (Alt+Insert) Button Pressed
To activate press spacebar.
Add New Configuration List view
Run External tool
1 of 7
To move to items use the Arrow keys.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![nvda is announcing some extra information when alt insert shortcut key is used in push image window](https://user-images.githubusercontent.com/22956130/36137608-075128be-10bc-11e8-991a-cec8c9be8af5.jpg)

**Comment by Eskibear** [comment]
Content hierarchy is provided by IntelliJ run/debug configuration, out of the toolkit's control.

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749942
